server/java/src/main/java/com/godaddy/asherah/grpc/AppEncryptionConfig.java

```diff
@@ -63,7 +66,37 @@ KeyManagementService setupKeyManagementService(final String kmsType, final Strin
 
       case Constants.METASTORE_DYNAMODB:
         logger.info("using DynamoDB-based metastore...");
-        return DynamoDbMetastoreImpl.newBuilder().build();
+        DynamoDbMetastoreImpl.Builder builder = DynamoDbMetastoreImpl.newBuilder();
+        String dynamoDbRegion = dynamoDbConfig.getDynamoDbRegion();
+        String keySuffix = dynamoDbConfig.getKeySuffix();
+        String tableName = dynamoDbConfig.getTableName();
+        String endPoint = dynamoDbConfig.getDynamoDbEndpointConfig();
+        String signingRegion = dynamoDbConfig.getDynamoDbSigningRegion();
+
+        // Check for region configuration
+        if (!StringUtils.isEmpty(dynamoDbRegion)) {
```

Comment:
I am able to start the server with `--dynamodb-region ''`
This shouldn't be allowed

Reply:
read comment above